spec/fixtures/addons/triangle.scss

```diff
@@ -0,0 +1,8 @@
+@import "setup";
+
+.triangle-down {
+  @include triangle(12px, gray, down);
+}
+.triangle-corner {
+  @include triangle(12px 6px, gray lavender, up-left);
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color `lavender` should be written in hexadecimal form as `#e6e6fa`